components/next_steps_view/steps/invite_members_step/invite_members_step.tsx

```diff
@@ -141,7 +136,8 @@ export default class InviteMembersStep extends React.PureComponent<Props, State>
     onBlur = () => {
         if (this.state.emailInput) {
             const emails = this.state.emailInput.split(/[\s,]+/).filter((email) => email.length).map((email) => ({label: email, value: email, error: !isEmail(email)}));
-            this.setState({emails: [...this.state.emails, ...emails], emailInput: '', emailError: undefined});
+            const newEmails = [...this.state.emails, ...emails];
+            this.setState({emails: newEmails, emailInput: '', emailError: newEmails.length > 10 ? Utils.localizeMessage('next_steps_view.invite_members_step.tooManyEmails', 'Invitations are limited to 10 email addresses.') : undefined});
```

Comment:
Actually, given that they already have the admin, it's should probably be limited to 9. Should we do the same we do in the invite modal and rely on the actual count of available users in the free tier?

Reply:
Seems kinda late to make this change to be honest, we should discuss it with the rest of the team first and I think it should be a separate ticket.

Reply:
Sure, could you create a follow up ticket?

Reply:
https://mattermost.atlassian.net/browse/MM-29958